src/test/scala/edu/berkeley/ce/rockslicing/BlockVTKSpec.scala

```diff
@@ -163,12 +165,12 @@ class BlockVTKSpec extends FunSuite {
   test("Connectivities for each face should be in terms of vertex ID's") {
     val vtkBlock = BlockVTK(unitCube)
     val faceConnectivities = Map(
-      face1 -> Seq[Int](7, 6, 5, 4),
-      face2 -> Seq[Int](3, 1, 0, 2),
-      face3 -> Seq[Int](5, 6, 0, 1),
-      face4 -> Seq[Int](7, 4, 3, 2),
-      face5 -> Seq[Int](2, 0, 6, 7),
-      face6 -> Seq[Int](4, 5, 1, 3)
+      face1 -> Seq[Int](7, 1, 3, 6),
```

Comment:
All these changes look annoying. Just curious, is there anything we can do to make these unit tests less brittle?

Obviously this wouldn't be something to add to this pull request, just wanted to hear your thoughts on this.